client/components/wpcom-colophon/README.md

```diff
@@ -0,0 +1,20 @@
+WpcomColophon (JSX)
+====================
+
+This component is used to display a "In partnership with WordPress.com" colophon, usually on the footer of page.
```

Comment:
Edit suggestions: 
> a "In partnership with WordPress.com" colophon
an "In partnership with WordPress.com" colophon?

> on the footer of page
"in a/the page footer"?

Reply:
👍 fixed in https://github.com/Automattic/wp-calypso/pull/30721/commits/3b6f51bc8ac99b4e6e75627cb480d01647046714